CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixed
 * Using valid const, but non-literal expressions, such as negation of numbers, as an initializer would fail. (Issue [#1606](https://github.com/realm/realm-dart/issues/1606))
+* Backlinks mistakenly included in EJson serialization ([Issue #1616](https://github.com/realm/realm-dart/issues/1616))
```

Comment:
Wondering if it's worth adding `since 2.0.0`? The current CHANGELOG pattern seems to only use `since` for Core related changes though.
```suggestion
* Backlinks mistakenly included in EJson serialization. ([Issue #1616](https://github.com/realm/realm-dart/issues/1616))
```

Reply:
We don't usually do that.  Also this bug has been there forever in the sense that it existed in the very first release supporting EJson.